doc/project/checklist.md

```diff
@@ -184,9 +184,10 @@ Prefix "SIM" is applicable for simulation-based DV approach only, while "FPV" is
 - DV Plan document drafted, indicating the overall DV strategy, intent and the testbench environment details with diagrams, details on TB, UVCs, checkers, scoreboard, interfaces, assertions, coverage objects (if applicable).
 - Details may be missing since most items are not expected to be fully developed at this stage.
 
-### TESTPLAN_COMPLETED
+### VERIFICATION_PLAN_COMPLETED
```

Comment:
This is too close to `DV_PLAN_COMPLETED`. IMO we should disambiguate the two. 
Perhaps retain the original `TESTPLAN_COMPLETED`, now that the description clarifies the intent? We are calling the HJson also as `*_testplan.hjson`. Feel like this would end up prompting several other things to change.